scala/CustomTypeStalinSort.scala

```diff
@@ -66,4 +66,6 @@ object CustomStalinSort extends App {
   }
 
   println(vegetableList.stalinSort)
-}
+
+  println(Seq(1, 3, 3, 5, 4).stalinSort)  //if you run it, the result of is List(1, 3, 5)
```

Comment:
Hey mate, as is described at the README, we expect that equal numbers were preserved, could you update your code to this approach?

Reply:
The expected result is (1, 3, 3, 5)

Reply:
Oh...thank you for your pointing out!!
I overlooked this condition.
I will update this code.